CONTRIBUTING.md

```diff
@@ -132,7 +162,7 @@ Also don't forget the documentation (reference documentation for features, javad
 
 ## Update the issue with the correct release information:
 
-If the change would result in behaviour in Narayana that is incompatible with the current release stream for Narayana, then the relevant JIRA issue `Fix Version/s` must be set to the `<version+1>.next`. For example if the root pom.xml is currently `<version>0.0.10.Final-SNAPSHOT</version>` then the JIRA issue should have a `Fix Version/s` of `1.next` added.
+If the change would result in behaviour in Narayana that is incompatible with the current release stream for Narayana, then the relevant JIRA issue `Fix Version/s` must be set to the `<version+1>.next`. For example if the root pom.xml is currently `<version>7.0.3.Final-SNAPSHOT</version>` then the JIRA issue should have a `Fix Version/s` of `8.next` added. When resolving the issue in JIRA we encourage the assignee to consider adding a release note describing the impact of the change.
```

Comment:
side note: lra actual version is 1.0.3.Final-SNAPSHOT but we are discussing about moving the issue tracker to GitHub issue (see https://narayana.zulipchat.com/#narrow/channel/323715-developers/topic/LRA.20should.20not.20require.20JBTMs) so I will probably update this version soon to update the issue creation process (Jira -> GH). Therefore I am approving the PR.